temba/channels/models.py

```diff
@@ -146,6 +146,11 @@ def has_attachment_support(self, channel):
         """
         return self.attachment_support
 
+    def get_quick_replies(self, **kwargs):
```

Comment:
In the future, other channels may support quick replies, so I thought that we already could implement this way, furthermore the methods to get quick replies would need to be static.